WikidataItem.py

```diff
@@ -1,93 +1,236 @@
 # -*- coding: utf-8 -*-
-import importer_utils as utils
+"""
+An object that represent a Wikidata item.
+
+This is a basic object used to construct
+a Wikidata item. It contains the basic functions
+to create statements, qualifiers and sources,
+as well as labels and descriptions with specified
+languages.
+
+The purpose of it is to serve as a base for a
+data-specific object that will turn some data
+into Wikidata objects. It can then be uploaded
+to Wikidata using the uploader script.
+"""
 from wikidataStuff.WikidataStuff import WikidataStuff as WDS
+from wikidataStuff import helpers as helpers
 import pywikibot
 
+import importer_utils as utils
 
 DATA_DIR = "data"
 
 
 class WikidataItem(object):
     """Basic data object for upload to Wikidata."""
 
+    def __init__(self, db_row_dict, repository, data_files, existing):
+        """
+        Initialize the data object.
+
+        :param db_row_dict: raw data from the data source
+        :param db_row_dict: Expected type: string
```

Comment:
Since I'm doing this from the bottom up, I've added this to the scope of the follow-up task below:

https://phabricator.wikimedia.org/T165936